app/graphql/mutations/review_create.rb

```diff
@@ -44,6 +44,22 @@ def resolve(
         @client = Client.find_by(email: client_email)
       end
 
+      if size == -1
```

Comment:
such a good idea to make nil useable again! I wonder why the fe cant send nil/nul values?